tools/concat_release_notes.py

```diff
@@ -0,0 +1,47 @@
+from packaging import version
```

Comment:
This script combines all the distinct release note files per-release into one single file before Sphinx runs.